tests/test_spatialdata.py

```diff
@@ -100,3 +101,57 @@ def test_sdatawidget_points(caplog, make_napari_viewer: Any):
         )
     assert widget._viewer.layers[1].metadata.get("adata").n_obs == 100000
     del sdata.points["many_points"]
+
+
+def test_layer_visibility(qtbot, make_napari_viewer: Any):
+    set_transformation(sdata.points[list(sdata.points.keys())[0]], Identity(), to_coordinate_system="other")
+    viewer = make_napari_viewer()
+    widget = SdataWidget(viewer, sdata)
+
+    # Click on `global` coordinate system
+    center_pos = get_center_pos_listitem(widget.coordinate_system_widget, "global")
+    click_list_widget_item(qtbot, widget.coordinate_system_widget, center_pos, "currentItemChanged")
+
+    # Load 2 layers, make 1 invisible
+    widget._onClick(list(sdata.points.keys())[0])
+    widget._onClick(list(sdata.labels.keys())[0])
+
+    points = viewer.layers[0]
+    labels = viewer.layers[1]
+
+    assert points.metadata["active_in_cs"]
+    assert labels.metadata["active_in_cs"]
```

Comment:
What is the intention of the test here? To test if they key `"active_in_cs"` is present in the metadata dictionary? If so, I think it should be `assert "active_in_cs" in points.metadata`